misc/scripts/deploy-tools.sh

```diff
@@ -176,6 +176,22 @@ function deployESXPost {
     fi
 }
 
+function deployESXInstallForUpgrade {
+    $SSH $TARGET $VIB_INSTALL -v $VIB_URL
+    if [ $? -ne 0 ]
+    then
+        log "deployESXInstall: Installation hit an error on $TARGET"
+        exit 2
+    fi
+}
+
+function deployesxForUpgrade {
```

Comment:
nit: `deployESXInstallForUpgrade` and `deployESXInstallForUpgrade` .. better to keep the case in sync.